.github/actions/deploy-klt-on-cluster/action.yml

```diff
@@ -2,16 +2,20 @@ name: "Deploy Keptn Lifecycle Toolkit on GH cluster"
 description: "Creates a Kind cluster and deploys Keptn Lifecycle Toolkit"
 inputs:
    cert-manager-version:
-     required: true
+     required: false
```

Comment:
why the cert-manager is not required anymore? Does the mutating webhook work then ?

Reply:
It's not required to provide a specific cert-manager version while invoking this composite action. 
If no version is provided, the default value will be used.

Reply:
perfect, thx :)